README-PYPI.md

```diff
@@ -25,27 +25,28 @@ You can check your current version with `python --version`.
 The exact upgrade process will depend on your environment and operating system.
 
 ```
-usage: dp-wizard [-h] [--public_csv CSV] [--private_csv CSV] [--contrib CONTRIB] [--demo]
+usage: dp-wizard [-h] [--demo] [--no_uploads]
```

Comment:
These CLI options we're being used, and they created the possibility for inconsistent states. (What happens if `--private_csv` and `--no_uploads` are specified together?) Rather than adding complex validation logic, just get rid of features no one needs.

Reply:
Considering this a bit more, `demo` and `no_uploads` really should be mutually exclusive. I'll fix this.